ratis-server-api/src/main/java/org/apache/ratis/server/RaftServer.java

```diff
@@ -77,7 +80,14 @@ default RaftPeer getPeer() {
 
     /** @return the {@link RaftGroup} for this division. */
     default RaftGroup getGroup() {
-      return RaftGroup.valueOf(getMemberId().getGroupId(), getRaftConf().getAllPeers());
+      Collection<RaftPeer> allFollowerPeers =
+          getRaftConf().getAllPeers(RaftProtos.RaftPeerRole.FOLLOWER);
+      Collection<RaftPeer> allListenerPeers =
+          getRaftConf().getAllPeers(RaftProtos.RaftPeerRole.LISTENER);
+      Iterable<RaftPeer> peers = Iterables.concat(allFollowerPeers, allListenerPeers);
+      return RaftGroup.valueOf(
+          getMemberId().getGroupId(),
+          Lists.newArrayList(peers));
```

Comment:
Change the `peers` parameter of `RaftGroup.valueOf(..)` to `Iterable<RaftPeer>`.  Then, we don't have to create a new array list.
```
    default RaftGroup getGroup() {
      final Collection<RaftPeer> followers = getRaftConf().getAllPeers(RaftPeerRole.FOLLOWER);
      final Collection<RaftPeer> listeners = getRaftConf().getAllPeers(RaftPeerRole.LISTENER);
      final Iterable<RaftPeer> peers = Iterables.concat(followers, listeners);
      return RaftGroup.valueOf(getMemberId().getGroupId(), peers);
    }
```

Reply:
fix